index.js

```diff
@@ -42,6 +42,7 @@ function NYC (config) {
   this._reportDir = config.reportDir || 'coverage'
   this._sourceMap = typeof config.sourceMap === 'boolean' ? config.sourceMap : true
   this._showProcessTree = config.showProcessTree || false
+  this._buildProcessTree = this._showProcessTree || config.buildProcessTree
```

Comment:
This is the only thing that confuses me... Is there a difference between `build` and `show` here? Is it just a preference for the name `--build-process-tree`?

Reply:
In order to show the process tree, you must also build it.  So `--show-process-tree` implies `--build-process-tree` as well.  But the reverse is not true, ie, you could build it without showing it.

Reply:
@JaKXz any objections if I merge this?